sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -48,13 +48,13 @@
 
 // Downsize factor for video stream
 #ifndef VIEWVIDEO_DOWNSIZE_FACTOR
-#define VIEWVIDEO_DOWNSIZE_FACTOR 4
+#define VIEWVIDEO_DOWNSIZE_FACTOR 1
```

Comment:
I don't get it, you increase the size of the images/video as you don't downsize it anymore... you also increased the quality...
In any case, this change needs to be reflected in the module xml